src/db/db.instance.ts

```diff
@@ -0,0 +1,62 @@
+class DbInstance {
+  private items: any[];
+  private idCounter: number;
+
+  constructor(items = []) {
+    this.items = items;
+    this.idCounter = 0;
+  }
+
+  private getItemIndex(id: number): number {
+    const index = this.items.findIndex((item) => item.id === id);
+    return index;
+  }
+
+  public getAll() {
+    return this.items;
+  }
+
+  public createNew<T>(payload: T) {
```

Comment:
As I think, we always create a new instance :)

```suggestion
  public create<T>(payload: T) {
```